### PR TITLE
Add functionality to add # of shares bought

### PR DESCRIPTION
**Pull request by iliketigers**

### CODE REVIEW

File: src/utilities.js [hunk 1]
```diff
@@ -47,17 +48,22 @@ export function deleteLotFromPortfolio(id) {
 // TODO: need to convert this to sum values
 export const getDataForChart = (ticker) => {
   const pricesJson = Constants.API_PRICES[ticker];
-  const xAxisLabels = [];
-  const yAxisLabels = [];
-  for (let i = 0; i < pricesJson.length; i++) {
-    xAxisLabels.push(pricesJson[i].date.split('T')[0]);
-    yAxisLabels.push(pricesJson[i].close);
-  }
+  if (pricesJson) {
```
Comment (kamahen) [inline src/utilities.js:51]: Some people prefer a short-circuit style here. It's a matter of taste.

`if (!pricesJson) return;`

and then you don't need to have `if (pricesJson)` section indented.

File: src/AddLot.js [hunk 2]
```diff
@@ -2,25 +2,44 @@ import React from 'react';
...
+
+  handleSharesChange = (event) => {
+    this.setState({ boughtShares: event.target.value });
   };
 
   handleSubmit = (event) => {
     event.preventDefault();
     this.props.submitted(true);
-    Utilities.addLotToPortfolio(this.state.value);
+    Utilities.addLotToPortfolio(
```
Comment (kamahen) [inline src/AddLot.js:18]: Curious why you have this very spread-out style. I prefer things a bit tighter, e.g.

`Utilities.addLotToPortfolio(this.state.tickerValue, this.state.boughtShares);`

all on one line. This allows taking in a larger chunk of code when reading it.

File: src/utilities.js (hunk 1 above)
Comment (iliketigers) [reply src/utilities.js:51]: done

File: src/AddLot.js (hunk 2 above)
Comment (iliketigers) [reply src/AddLot.js:18]: just auto-formatting :) line length.

**Pull request merged**